template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -210,15 +224,18 @@ def check_uid_uniqueness(proof_dir, proof_uids):
     sys.exit(1)
 
 
-async def configure_proof_dirs(queue, counter, proof_uids):
+async def configure_proof_dirs(queue, counter, proof_uids, enable_pools):
     while True:
         print_counter(counter)
         path = str(await queue.get())
 
         check_uid_uniqueness(path, proof_uids)
 
+        pools = ["ENABLE_POOLS=true"] if enable_pools else []
```

Comment:
I've added a command-line option that allows you to turn the pools feature off. By default, it will be switched on iff Litani is detected to support it, but users may force it to be used or not used using the flag. See "Make parallelism of EXPENSIVE proofs configurable".